source/faucet/main.d

```diff
@@ -36,7 +36,7 @@ import dyaml.loader;
 
 import std.algorithm;
 import std.exception;
-static import std.file;
+import std.file;
```

Comment:
I guess we don't need `static` ?

Reply:
Not sure why it was `static import` to start with but I wanted to use `.exists` after the name of the config file so it would not work as it was.

Reply:
I usually `static import std.file` because the names it provides are very generic. But it's a personal habit, so I don't mind changing it.